service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -343,26 +353,22 @@ private ClientConfig buildClientConfig() {
     return builder.build();
   }
 
-  @VisibleForTesting
-  public ClientConfig getClientConfig() {
-    assertCrlInUse();
-    return clientConfig;
-  }
-
-  private void assertCrlInUse() {
-    if (!crlConfig.getUseCrl()) {
-      throw new CrlNotInUseException("Attempt to use CRL when it is set not to be used");
-    }
-  }
-
-  // -- Azure Support --
+  //  Azure Support
   // TODO: this is not in CRL, but it will be, so this is a convenient place to dump the
```

Comment:
Not your comment, but CRL doesn't actually wrap the `TokenCredential` class, so clients still need to instantiate one with the appropriate clientId/secret/tenant.